packages/ember-htmlbars/lib/keywords/mount.js

```diff
@@ -0,0 +1,207 @@
+/**
+@module ember
+@submodule ember-templates
+*/
+
+import { read } from 'ember-htmlbars/streams/utils';
+import ViewNodeManager from 'ember-htmlbars/node-managers/view-node-manager';
+import RenderEnv from 'ember-htmlbars/system/render-env';
+import EmptyObject from 'ember-metal/empty_object';
+import { assert } from 'ember-metal/debug';
+import { getOwner, setOwner } from 'container/owner';
+
+/**
+  The `{{mount}}` helper lets you embed a routeless engine in a template.
+
+  Mounting an engine will cause an instance to be booted and its `application`
+  template to be rendered.
+
+  For example, the following template mounts the `ember-chat` engine:
+
+  ```handlebars
+  {{! application.hbs }}
+  {{mount "ember-chat"}}
+  ```
+
+  Currently, the engine name is the only argument that can be passed to
+  `{{mount}}`.
+
+  @method mount
+  @for Ember.Templates.helpers
+  @category ember-application-engines
+  @public
+*/
+export default {
+  willRender(renderNode, env) {
+    if (env.view.ownerView._outlets) {
```

Comment:
This guard seems to be necessary for the `mount` tests in this PR to pass. Curiously, it is no longer required in the addon's tests (on `master`, anyway).